### PR TITLE
Replacing local playbook.yml with ad-hoc previewer command

### PR DESCRIPTION
**Pull request by akosma**
This PR replaces the local `playbook.yml` file with an ad-hoc previewer command based on a Docker image.

Developers can still `make docs` and preview their documentation in http://localhost:2020.

The `vale` related functionality is left unchanged.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,17 +5,11 @@ VERSION ?= $(shell git describe --tags --always --dirty --match=v* || (echo "com
...
-pages   := $(shell find . -type f -name '*.adoc')
-web_dir := ./_antora
-
 docker_cmd  ?= docker
 docker_opts ?= --rm --tty --user "$$(id -u)"
 
-antora_cmd  ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}":/antora vshn/antora:2.3.0
-antora_opts ?= --cache-dir=.cache/antora
-
 vale_cmd ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}"/docs/modules/ROOT/pages:/pages vshn/vale:2.1.1 --minAlertLevel=error --config=/pages/.vale.ini /pages
```
Comment (corvus-ch) [inline Makefile:11]: Separate variables by empty lines or align the assignment.

```suggestion
vale_cmd           ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}"/docs/modules/ROOT/pages:/pages vshn/vale:2.1.1 --minAlertLevel=error --config=/pages/.vale.ini /pages
```

**Review by corvus-ch — APPROVED** [review]
Nice one. Thanks

**Comment by tobru** [comment]
Great effort, here are a few comments:

* The `Makefile` is a generic one for the whole project, not just for docs. So I would suggest to prefix all commands which are doing stuff for documentation with `docs-`. For example `make docs-serve`, `make docs-vale`, ... This way it's very clear that these commands are solely for working with documentation stuff.
* `make docs-open` would be cool to have, calling either `open` or `xdg-open`, like discussed in the chat
  * Also do an `echo "Documentation is available under http://localhost:2020/"` when starting up the container, this way one can also just click on the link as most modern terminals discover URLs and make them clickable - as an alternative to `make docs-open`
* Move the code for the image `vshn/antora-preview` to GitHub and make sure it has an appropriate license
* All Project Syn tools should have a similar `Makefile`, all doing the same when calling `make docs-XYZ`. I haven't checked that in all the current open PRs though, but until now the commands weren't aligned. We might even make sure that all Antora based repositories are having the same `make` commands available, this helps with the muscle memory
* https://github.com/appuio/openshift4-docs/blob/master/Makefile -> This one also has support for Podman, running the container in rootless mode. Would be cool to also have this on all other Makefiles, again to make all the Antora based projects as similar as possible.

Let's discuss and address this points here, before we do a mass update of all other Antora based repos.

**Comment by srueg** [comment]
Maybe we could add all these `docs-*` targets to a `antora.mk` file to include in the main `Makefile`?
I fear that otherwise the main Makefile gets really bloated. It would also help to keep them in sync (i.e. also with module-sync at some point).

**Comment by akosma** [comment]
Well the harmonization of all Makefiles with the same style will be quite a bit of work, as some are use UPPERCASE all over the place, others not… etc. This should probably be a separate task?

For my part, I will update all the PRs with `docs-serve` and `docs-vale` tasks, and will move the project to Github.

**Comment by akosma** [comment]
Oh, and add a message in the `antora-preview` Docker image to remind the user to open a browser.

**Comment by akosma** [comment]
- Moved the code of the image to [Github](https://github.com/vshn/antora-preview)
- Uniform Makefile targets (`docs-serve` and `docs-vale`) in all PRs:
   - https://github.com/projectsyn/lieutenant-operator/pull/100
   - https://github.com/projectsyn/lieutenant-api/pull/79
   - https://github.com/projectsyn/commodore/pull/165
   - https://github.com/projectsyn/documentation/pull/85
   - https://github.com/projectsyn/steward/pull/48 (this one)

**Pull request merged**